### DISCUSSION TITLE
Update from 9.16.2 to 9.17.0 resulted in lost of half of the zwave network.

### DISCUSSION THREAD

**Discussion by Judman** [zwave-js/zwave-js#7093] [Request Support / Investigate Issue]
Hello,

Last night I updated from 9.16.2 to 9.17.0. When the update was complete only 10 of my 20 devices are showing on the network.

I restored the 9.16.2 backup HA made, but that did not fix the problem.

I am using HA version 2024.7.4 with zwavejs and zwavejs-iu as addons. I am using a Zooz 800 Series GPIO Module over Ethernet in a Olimex ESP32-PoE with ESPHome firmware , FW v1.20 SDK v7.19.3.  https://tubeszb.com/product/z-wave-poe-kit/

Attached are the logs I have and node config. I really do not know where to even start troubleshooting. I was thinking I might try to restore a proxmox backup from 4 days ago, but if the issues is with the controller it would be a waste.

[zwavejs_2024-08-08.log](https://github.com/user-attachments/files/16561031/zwavejs_2024-08-08.log)
[nodes.json](https://github.com/user-attachments/files/16561037/nodes.json)
[settings.json](https://github.com/user-attachments/files/16561038/settings.json)
[zwavejs_current.log](https://github.com/user-attachments/files/16561039/zwavejs_current.log)
[zwavejs_current.log](https://github.com/user-attachments/files/16561043/zwavejs_current.log)

![image](https://github.com/user-attachments/assets/fbf4dd57-c3be-4db3-8937-310a6f9029c7)
![image](https://github.com/user-attachments/assets/2f183517-49c3-4ee2-adcf-ca514fd7b4f5)

**Comment by Judman** [comment]
@robertsLando I wasn't sure where to post this. I figured since updating the zwaveju-ui seems to cause the issue that was the right place. My Apologies!

**Comment by robertsLando** [comment]
@Judman No worries :)

**Comment by Judman** [comment]
Here is another log showing a restart before the update with all nodes recognized.

https://pastebin.com/bcKEEbSz

**Comment by AlCalzone** [comment]
Looks like the controller "forgot" about the nodes for some reason. No idea how that can even happen, other than cutting power right when the controller was re-writing its memory.

If you have an NVM backup from before it happened, you can try restoring that.
If not, you can make a new one, send it to me and I can take a look if the information is still recoverable.

**Reply by Judman** [reply]
@AlCalzone Thanks for the response. A power cut would have been a crazy fluke. All that is powered from a UPS so maybe the switch or esp had a hiccup.

Attached is a backup from my Z-wave.me controller I replaced back in May with the ZAC93. I also made a backup after the issue took place. I had to zip them since Github doesn't accept the bin files.

[NVM_20240528224442.zip](https://github.com/user-attachments/files/16581204/NVM_20240528224442.zip)
[NVM_20240809134047.zip](https://github.com/user-attachments/files/16581215/NVM_20240809134047.zip)

I did restore the 05.28 backup to the controller and it seems everything is okay at the moment. I will have to re-include a few devices that were added after the backup.

**Reply by AlCalzone** [reply]
I finally managed to look at the NVM. It looks like one node after another was removed from the network. I can't see if this happened over a longer timespan, or "instantly" though.

**Comment by Judman** [comment]
No real cause has been identified. No idea if it was a hardware or software issue. Closing topic because everything has been working okay since the restore.
